metadata_backend/helpers/validator.py

```diff
@@ -76,3 +80,75 @@ def is_valid(self) -> bool:
         """Quick method for checking validation result."""
         resp = json.loads(self.resp_body)
         return resp['isValid']
+
+
+def extend_with_default(validator_class: Draft7Validator) -> Draft7Validator:
```

Comment:
why do we do this, I thought we don't want to use default values in jsonschemas?

Reply:
in case they are added  I do extra or will be added at some point for any reason e.g. custom schemas.